www/templates/default/html/js/manager.js

```diff
@@ -1,3 +1,7 @@
+require(['jquery', frontend_url + 'templates/default/html/js/vendor/select2/js/select2.min.js'], function($) {
+	$(".use-select2").select2();
```

Comment:
fixed in https://github.com/mfairchild365/UNL_UCBCN_System/commit/b291b01aac018db0f895a19a4ea6d06320340d00